src/components/ShareStructure/ShareStructure.vue

```diff
@@ -318,26 +340,36 @@
 
 <script lang="ts">
 // Libraries
-import { Component, Emit, Vue, Watch } from 'vue-property-decorator'
+import { Component, Mixins, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'vuex-class'
 import 'array.prototype.move'
-import { isEqual, omit } from 'lodash'
+import { cloneDeep, isEqual, omit } from 'lodash'
 
 // Components
 import { ActionChip } from '@/components/common'
 import EditShareStructure from './EditShareStructure.vue'
+import * as Dialogs from '@/components/dialogs'
+
+// Mixins
+import { CommonMixin } from '@/mixins'
 
 // Interfaces or Enums
-import { ActionBindingIF, IncorporationFilingIF, ShareClassIF } from '@/interfaces'
 import { ActionTypes } from '@/enums'
+import { ActionBindingIF, ConfirmDialogType, IncorporationFilingIF, ShareClassIF } from '@/interfaces'
 
 @Component({
   components: {
     ActionChip,
-    EditShareStructure
+    EditShareStructure,
+    ...Dialogs
```

Comment:
I think this will import/use all dialogs (which makes sense for App.vue) but maybe here you only want to import the one you use?

Reply:
Updated!